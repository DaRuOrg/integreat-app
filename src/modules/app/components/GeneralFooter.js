import React from 'react'
import { translate } from 'react-i18next'
import Footer from 'modules/layout/components/Footer'

class GeneralFooter extends React.Component {
  render () {
    return <Footer items={[{href: '/', text: this.props.t('imprintAndContact')}]} />
  }
}

export default translate('app')(GeneralFooter)
