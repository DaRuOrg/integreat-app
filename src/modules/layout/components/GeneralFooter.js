import React from 'react'
import { translate } from 'react-i18next'
import PropTypes from 'prop-types'

import Footer from './Footer'

class GeneralFooter extends React.Component {
  static propTypes = {
    t: PropTypes.func.isRequired
  }
  render () {
    return <Footer items={[{href: '/disclaimer', text: this.props.t('imprintAndContact')}]} />
  }
}

export default translate('app')(GeneralFooter)
