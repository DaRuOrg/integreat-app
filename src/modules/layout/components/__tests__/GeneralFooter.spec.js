import { shallow } from 'enzyme'
import React from 'react'
import GeneralFooter from '../GeneralFooter'

describe('GeneralFooter', () => {
  test('should match snapshot', () => {
    const component = shallow(<GeneralFooter />)
    expect(component.dive()).toMatchSnapshot()
  })
})
