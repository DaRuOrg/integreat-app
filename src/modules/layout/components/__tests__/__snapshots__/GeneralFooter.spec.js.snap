// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneralFooter should match snapshot 1`] = `
<Footer>
  <Connect(Link)
    href="/disclaimer"
  >
    app:imprintAndContact
  </Connect(Link)>
  <a
    href="https://integreat-app.de/datenschutz/"
  >
    Datenschutz
  </a>
</Footer>
`;
